test/FunctionalTests/Tests/DataServices/UnitTests/DesignT4UnitTests/ODataT4CodeGeneratorTestDescriptors.cs

```diff
@@ -46,6 +49,25 @@ public class ODataT4CodeGeneratorTestsDescriptor
             public Action<string, bool, bool> Verify { get; set; }
         }
 
+
+        internal static void ValidateXMLFile(string tempFilePath)

```

Comment:
Any reason why `ValidateXMLFile` and `ValidateEdmx` are internal instead of private?

Reply:
they can be reused in the project

Reply:
Seems unnecessary at this point as there's no case for reuse